imports/server/git_revision.ts

```diff
@@ -1,14 +0,0 @@
-import { Meteor } from 'meteor/meteor';
```

Comment:
Don't we need to remove the import of this from `server/main.ts`?

Reply:
...yes and I'm sure the tests will momentarily remind me of that. 🤦🏻